podcasts/Profile - SwiftUI/Account/AccountHeaderView.swift

```diff
@@ -28,21 +30,32 @@ struct AccountHeaderView: View {
                         Text(title)
                             .fixedSize(horizontal: false, vertical: true)
                         Spacer()
-                        Button(action: {
-                            Toast.show(L10n.plusChampionMessage)
-                        }, label: {
-                            label
-                        })
+                        label
+                        .onTapGesture {
+                            action?()
+                        }
                             .fixedSize(horizontal: false, vertical: true)
                     }
                     .foregroundColor(theme.primaryText01)
                     .font(size: 14, style: .subheadline, weight: .medium)
+                    .sheet(isPresented: $showingChampion) {
+                        ZStack {
+                            theme.primaryUi01.ignoresSafeArea()
+
+                            if #available(iOS 16.0, *) {
+                                ChampionView()
+                                    .presentationDetents([.medium])
+                            } else {
+                                ChampionView()
```

Comment:
The iOS 15 result will open on a big screen. I think this is an ok compromise as ~2% of users are on it and this is not a core flow/screen.

Reply:
It makes sense as making a host vc just to set the detents doesn't make sense